specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1910,6 +2067,255 @@
           "description": "A textual explanation that can be used for display purposes to summarize the index value and category. For example, when the index value for Flight Delays is very good, the description will be \"Conditions are excellent for flying!\"."
         }
       }
+    },
+    "CurrentAirQualityResponse": {
+      "type": "object",
+      "description": "This object is returned from a successful Get Current Air Quality call.",
+      "properties": {
+        "results": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AirQuality"
+          },
+          "description": "A list of all air quality results for the queried location."
+        }
+      }
+    },
+    "HourlyAirQualityForecastResponse": {
+      "type": "object",
+      "description": "This object is returned from a successful Get Hourly Air Quality Forecast call.",
+      "properties": {
+        "forecasts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AirQuality"
+          },
+          "description": "A list of all hourly air quality forecasts for the queried location."
+        }
+      }
+    },
+    "DailyAirQualityForecastResponse": {
+      "type": "object",
+      "description": "This object is returned from a successful Get Daily Air Quality Forecast call.",
+      "properties": {
+        "forecasts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DailyAirQuality"
+          },
+          "description": "A list of all daily air quality forecasts for the queried location."
+        }
+      }
+    },
+    "AirQuality": {
+      "type": "object",
+      "description": "Information about the air quality in a specific location at a specific time.",
+      "properties": {
+        "dateTime": {
+          "type": "string",
```

Comment:
`"format": "date-time"` will be a good addition here as the proceeding codegen will result in date types rather then raw strings:  https://github.com/ambientlight/azure-sdk-for-go/blob/825aa43364c936dfde70094d465ab8e7e07eef10/services/preview/maps/1.0/weather/models.go#L45

Reply:
Updated